### DISCUSSION TITLE
Excitement penalty for the Lay-down Coaster based on too little flying track

### DISCUSSION THREAD

**Discussion by visor841** [OpenRCT2/OpenRCT2#17419] [Feature requests] [answered]
For a scenario, I built a Lay-down Coaster with decent stats that looked interesting, and posted it to Reddit. Someone pointed out that it would actually be very uncomfortable to ride because the riders are laying on their back for pretty much the entire ride. Instead the ride is best enjoyed with lots of time spent in the "flying" position, face down, i.e. inverted. Would it make sense/be possible to give an excitement penalty if not enough of the Lay-down Coaster track is inverted? Perhaps divided by 2 if less than half of the track is inverted/inversions, and divided by 2 again if less than a quarter of the track is inverted/inversions. Perhaps inversions could count as half inverted?

I'd be willing to try to do it myself if the idea is accepted.

**Comment by 733737** [comment, marked as answer]
The team of developers probably will not change this because similar requests have been denied in the past simply because it is against the goals of the OpenRCT2 team

**Reply by IntelOrca** [reply]
It isn't against the goals. Changing the ratings requires a lot of thought and due diligence, otherwise a lot of players will be wondering why the ratings in their parks and coasters have changed so much.

Ratings have already been changed, usually to be more relaxed.
